src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,202 @@
+package duke.parser;
+
+import duke.command.Command;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.InvalidCommand;
+import duke.command.EventCommand;
+import duke.command.DeadlineCommand;
+import duke.command.TodoCommand;
+import duke.command.DeleteCommand;
+import duke.command.DoneCommand;
+import duke.command.ListCommand;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Parses user input.
+ */
+public class Parser {
```

Comment:
consider renaming your class to a `Processor` or something else. to me, it feels like a Parser should just return a parsed String, but not return a Command object.